frappe/utils/backups.py

```diff
@@ -636,6 +671,61 @@ def backup(
 		"backup_path_private_files": odb.backup_path_private_files,
 	}
 
+@frappe.whitelist()
+def get_backup_encryption_key():
+	if 'backup_encryption_key' in frappe.local.conf:
+		message = frappe.local.conf.backup_encryption_key
+	else:
+		message = "No key found."
+	return frappe.msgprint(message,'Backup Encryption Key')
```

Comment:
API endpoints should respond with some specific data, just returning `frappe.conf.backup_encryption_key` would do here. The user message should be handled at the frontend.

Also, this API should have a restriction such as only for System Managers, or the Administrator user.